apps/etterlatte-behandling/src/main/kotlin/oppgave/OppgaveDao.kt

```diff
@@ -570,6 +610,10 @@ class OppgaveDaoImpl(private val connectionAutoclosing: ConnectionAutoclosing) :
                 getString("saksbehandler")?.let {
                     OppgaveSaksbehandler(getString("saksbehandler"), getString("navn"))
                 },
+            forrigeSaksbehandler =
+                getString("forrige_saksbehandler")?.let {
+                    OppgaveSaksbehandler(getString("forrige_saksbehandler"), getString("navn"))
```

Comment:
Navn for forrige saksbehandler blir her satt som navnet til nåværende saksbehandler? Hvis den ikke trengs tror jeg det er bedre å sette både navn og ident til å være ident, i stedet for noe som blir til feil person

Reply:
Evt. bare kalle feltet `forrigeSaksbehandlerIdent` i `OppgaveIntern`, og ikke dytte inn saksbehandler med navn

Reply:
Ja det gir kanskje mer mening ja